pkg/specgen/generate/container.go

```diff
@@ -169,6 +170,19 @@ func CompleteSpec(ctx context.Context, r *libpod.Runtime, s *specgen.SpecGenerat
 		}
 	}
 
+	// If caller did not specify Pids Limits load default
+	if s.ResourceLimits.Pids == nil {
```

Comment:
This could null-pointer, I think - `s.ResourceLimits` may be null.

Reply:
Fixed